From a59d5c8b46291c24de1302dcdea2c1b3add8497c Mon Sep 17 00:00:00 2001
From: ejaaskel <esa.jaaskela@suomi24.fi>
Date: Tue, 3 Mar 2026 18:44:23 +0000
Subject: [PATCH 1/2] policy/modules/services/ssh: Allow ssh_sysadm_login

Without ssh_sysadm_login tunable the SSH logins seem to fail. The
following error can be seen when using SSH command:

/bin/sh: Permission denied

From the audit logs following issue can be seen:

avc:  denied  { transition } for  pid=462 comm="sshd" path="/bin/bash.bash" dev="vda" ino=16 scontext=system_u:system_r:sshd_t:s0 tcontext=unconfined_u:unconfined_r:unconfined_t:s0 tclass=process permissive=0
---
 policy/modules/services/ssh.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/modules/services/ssh.te b/policy/modules/services/ssh.te
index 8c9e61901..818be318e 100644
--- a/policy/modules/services/ssh.te
+++ b/policy/modules/services/ssh.te
@@ -21,7 +21,7 @@ gen_tunable(allow_ssh_keysign, false)
 ## Allow ssh logins as sysadm_r:sysadm_t
 ## </p>
 ## </desc>
-gen_tunable(ssh_sysadm_login, false)
+gen_tunable(ssh_sysadm_login, true)
 
 ## <desc>
 ## <p>
-- 
2.34.1

